src/screens/Profile/index.js

```diff
@@ -0,0 +1,14 @@
+import {StyleSheet, Text, View} from 'react-native';
+import React from 'react';
+
+import styles from './styles';
+
+const Profile = () => {
+  return (
+    <View style={styles.profile_container}>
+      <Text style={styles.profile_text}>Profile</Text>
```

Comment:
Create one constant folder move the text profile and every text then import here